themes/wilson_theme_starterkit/.eslintrc.json

```diff
@@ -11,7 +11,8 @@
   },
   "globals": {
     "Drupal": true,
-    "drupalSettings": true
+    "drupalSettings": true,
+    "once": true
```

Comment:
oh... OH... Is this how you fix the error that `Drupal is not defined` when you're using it in a script??? I could NOT figure out how to do this!

Reply:
I was stuck on that for a while myself till I found this.